main_test.py

```diff
@@ -7,137 +7,7 @@
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "G" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "H" }
-            ],
-            [
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "A" },
-                { "period": "B", "room": "TMAC-007", "name": "Evolution of Society: Economics & Technology", "teacher_username": "mdelaney", "department": "Interdisciplinary Electives" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "C" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "D" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "E" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "F" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "G" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "H" }
-            ],
-            [
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "A" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "B" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "C" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "D" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "E" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "F" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "G" },
-                { "room": None, "name": "Free Period", "teacher": None, "teacher_username": None, "department": None, "period": "H" }
-            ]
-        ],
-        "sid": 106,
-        "nickname": None,
-        "firstname": "Jonathan",
-        "lastname": "Briggs",
-        "gradyear": None,
-        "username": "jbriggs",
-        "advisor": None,
-        "grade": None
-    },
-}
-
+TEST_SCHEDULES = json.load(open('data/test_schedule.json'))

```

Comment:
After loading files, we really ought to close them - this is what the `with` keyword is for.